src/main/java/models/Phone.java

```diff
@@ -0,0 +1,13 @@
+package models;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
```

Comment:
if you use final for fields, you don't need allArgs, you need to use requiredArgsConstr.  As requiredArgs is already in @data annotation you can just remove allArgs. https://projectlombok.org/features/Data
Or you can use `record`